### PR TITLE
Dx 496

### PR DESCRIPTION
**Pull request by jmulford-bw**
## For the Committer

⚠️ Ensure that for this repo (**bandwidth.github.io**) that the pull request change is opened against the branch `gitbook` 

- [ ] I made sure that the site looks great and functions perfectly live
- [ ] I ran splell check
- [ ] I'm ready for these changes to be made live

### General Review

- [ ] All changes/updates requested were intentionally changed or added (no extraneous file or partial updates)
- [ ] The live rendered page behaves and looks like intended.
- [ ] All links work as intended. (click every link to make sure that it takes the user to the expected place).
- [ ] The markdown was rendered to HTML as intended. (Tables look right, code samples are gated properly).
- [ ] Any style or CSS changes are tested on multiple different pages to ensure nothing unexpected broke or changed.
- [ ] I proof-read the live site (`gitbook serve`) and there are no oblivious splelling or grammer misteaks.

### Gitbook Specific

- [ ] Any new documentation pages have been added to the `SUMMARY.md` file so they are rendered to HTML.
- [ ] Any links to other pages within this documentation use relative links (`[read more about ...](guides/voice/voicemail.md)`).
- [ ] Gitbook can install and build the documentation (`gitbook install` & `gitbook build`)

### Code Specific

- [ ] All code has been tested against the live API.
- [ ] All code can be run without throwing errors or **new** warnings.
- [ ] All code is formatted correctly and consistently (spaces, tabs).
- [ ] All code is legible and idiomatic to the language.
- [ ] All code uses sensical method, function, and variable names.
- [ ] All code samples use the appropriate syntax highlighting.
- [ ] Any examples re-use same phone numbers, uuids, variable names, etc... throughout (IE don't have something like `{from: '+19191231234', to: '+19191231234'}` unless it's meant to show the same phone number calling/texting itself ).

## TL;DR

- [ ] I did these things
- [ ] I'm ready for these changes to be made live

### CODE REVIEW

File: SUMMARY.md [hunk 1]
```diff
@@ -142,11 +142,3 @@
 	* [403 - Forbidden](messaging/errors/forbidden.md)
 	* [415 - Unsupported Content Type](messaging/errors/unsupportedContentType.md)
 	* [429 - Too Many Requests](messaging/errors/tooManyReq.md)
-
-## SDKs
```
Comment (dtolb) [inline SUMMARY.md]: We'll need to keep these in the short term. Eventually they will link to the page that has a blurb about how to install the "single" SDK. Let's keep these pages and just add a place holder `csharp.md` , `java.md`, etc...

File: applications/about.md [hunk 2]
```diff
@@ -1,11 +1,9 @@
 # Applications
 
-Bandwidth uses HTTP Callbacks (also known as [webhooks](https://webhooks.pbworks.com/w/page/13385124/FrontPage)) to send [message events](../events/messageEvents.md) to any publicly addressable url.
+Bandwidth uses HTTP Callbacks (also known as [webhooks](https://webhooks.pbworks.com/w/page/13385124/FrontPage)) to send [message events](../messaging/callbacks/messageEvents.md) to any publicly addressable url.
 
 In order to successfully use the Messaging 2.0 APIs, you need to configure an `Application` which contains the `CallbackUrl`.  Each application can be assigned to as many `Locations` as needed.  However, each `Location` can only have a single `Application`
 
-![Application Diagram](../images/messaging-2/applications-flow-messaging.png)
```
Comment (dtolb) [inline applications/about.md]: We'll need an updated graphic from UX

File: messaging/about.md [hunk 3]
```diff
@@ -25,9 +25,9 @@ There are a few concepts that are important to understand how Bandwidth's new me
 
 ## Message Storage IE `GET /messages` {#manage-storage}
 
-Messaging 2.0 does not keep _any_ records to fetch later.  If you need to keep track of delivered, error-ed, received messages after you receive the corresponding [callback event](events/messageEvents.md), you **MUST** store the events in the data-store of your choice.
+Messaging 2.0 does not keep _any_ records to fetch later.  If you need to keep track of delivered, error-ed, received messages after you receive the corresponding callback event, you **MUST** store the events in the data-store of your choice.
```
Comment (dtolb) [inline messaging/about.md]: Let's just say: "Bandwidth Messaging"

File: messaging/about.md [hunk 4]
```diff
@@ -37,11 +37,11 @@ After 24 hours, if your server has not returned a `HTTP 2xx` code, Bandwidth wil
 
 ## Message Creation/Acceptance (`HTTP 201` vs `HTTP 202`) {#message-creation}
 
-The messaging 2.0 API works off of an internal queuing system.  As such, when you <code class="post">POST</code> to the `v2/.../messages` to [create a new message](methods/sendMessages.md), Bandwidth will reply with an `HTTP 202 - Accepted`.  This indicates that the message has been placed on the queue
+The messaging 2.0 API works off of an internal queuing system.  As such, when you <code class="post">POST</code> to the `v2/.../messages` to create a new message, Bandwidth will reply with an `HTTP 202 - Accepted`.  This indicates that the message has been placed on the queue
```
Comment (dtolb) [inline messaging/about.md]: Same as above... The "messaging api"

File: messaging/ratelimits.md [hunk 5]
```diff
@@ -4,15 +4,6 @@
 
 This guide will walk through the recommended approach to managing queues and rate limits for use with the Messaging API. Over the past years mobile telecom operators have begun to block what is deemed automated traffic (**A2P**) sent over standard local telephone numbers IE: (919)-430-5555. The amount of messages sent in this way have increased due to spreading automated traffic (A2P) across multiple local telephone numbers to bypass volumetric filters. This process is called "snowshoeing" and as a result, the mobile operators are not only blocking volumetrically, but are also finger-printing content and preemptively blocking messages even from a "fresh" phone number.
 
-## Assumptions
```
Comment (dtolb) [inline messaging/ratelimits.md]: Let's leave the assumptions about CTIA & TCPA. those are frequently sent in sales as they govern how we are able to operate.

**Review by dtolb — COMMENTED** [review]
Let's keep the `sdks/{language}.md` files as a place holder for what we will eventually put there. It's ok if the page is blank!

Good work

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-DX-496.s3-website-us-east-1.amazonaws.com/

File: SUMMARY.md (hunk 1 above)
Comment (jmulford-bandwidth) [reply SUMMARY.md]: Done!

File: messaging/about.md (hunk 3 above)
Comment (jmulford-bandwidth) [reply messaging/about.md]: Good catch. Let's keep an eye out for more occurrences of these

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-DX-496.s3-website-us-east-1.amazonaws.com/

File: messaging/ratelimits.md (hunk 5 above)
Comment (jmulford-bandwidth) [reply messaging/ratelimits.md]: Sounds good. Changed

File: messaging/about.md (hunk 4 above)
Comment (jmulford-bandwidth) [reply messaging/about.md]: Fixed

File: applications/about.md (hunk 2 above)
Comment (jmulford-bandwidth) [reply applications/about.md]: Yes. I didn't see a link to an appropriate graphic

**Comment by jmulford-bandwidth** [comment]
@dtolb I went through all of your feedback. Let me know if there's anything else to address!

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-DX-496.s3-website-us-east-1.amazonaws.com/

**Pull request merged**